.pre-commit-config.yaml

```diff
@@ -0,0 +1,55 @@
+ci:
+  autofix_prs: false
+
+repos:
+  # Lint first (with safe autofixes)
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.6.9
+    hooks:
+      - id: ruff
+        name: ruff-check
+        args: [--fix, --show-fixes]
```

Comment:
Can you show an example with and without `--show-fixes` here? Is there a reason you wanted that here?

Reply:
--show-fixes makes Ruff print out which fixes it applied when run with --fix. I included it to make CI logs more transparent about what’s being changed, but I’m fine removing it if we’d prefer leaner logs. I can paste an example

Reply:
without `--show-fixes`
```
$ ruff check bad.py --fix
bad.py:1:1: F401 `os` imported but unused
```
with `--show-fixes`
```
$ ruff check bad.py --fix --show-fixes
bad.py:1:1: F401 `os` imported but unused
  Autofix: remove unused import
```

Reply:
I think we had this in the ilab cli